src/Npc.js

```diff
@@ -26,7 +26,7 @@ class Npc extends Character {
     this.area = data.area;
     this.behaviors = new Map(Object.entries(data.behaviors || {}));
     this.damage = data.damage;
-    this.defaultEquipment = data.equipment || [];
+    this.defaultEquipment = data.defaultEquipment || [];
```

Comment:
```suggestion
     this.defaultEquipment = data.equipment || {};
```

`data.equipment` was correct. The change should be that the default is an object `{}` instead of an array. It was looking for this in the YAML

```yaml
equipment:
  head: somearea:someitem
```

Reply:
ahhh, okay.  That makes sense now!